app/src/main/AndroidManifest.xml

```diff
@@ -5,6 +5,7 @@
 
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+    <uses-permission android:name="android.permission.WAKE_LOCK" />
```

Comment:
@thagikura Sorry, I overlooked your comment 🙇 
As you said, it looks unnecessary.

@kobakei If you have a time, could you please send PR again 🙏  Thanks!